crowbar_framework/config/repos.yml

```diff
@@ -5,169 +5,121 @@ suse-12.0:
       name: "SLES12-Pool"
       required: "mandatory"
       product_name: "common"
-      content_key:
```

Comment:
First, nothing validating the value of these attributes exists, so it's useless. Second, the etherpad marks this as optional, so it's actually fine to define repos without this attribute.